App.News/NewsManager.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using App.News.Repositories;
+using System.Linq;
+using App.News.Models;
+using App.Configuration;
+using App.News.Interfaces;
+using System.Collections.Generic;
+
+namespace App.News
+{
+    public class NewsManager: INewsManager,ITransientDependency
+    {
+        private readonly INewsRepository newsRepository; 
+        public NewsManager(INewsRepository newsRepository)
+        {
+            this.newsRepository = newsRepository;
+        }
+       
+        public string AddComment(CommentDTO comment)
+        {
+            var news = newsRepository.GetNewsById(comment.NewsId);
+            if (news == null)
+            {
+                return "news not found";
+            }
+            news.Comments.ToList().Add(comment);
+            return "ok";
+        }
+
+        public IEnumerable<CommentDTO> GetNewsComments(int newsId)
+        {
+            return newsRepository.GetNewsById(newsId).Comments;
+        }
```

Comment:
I guess this code responsive for working with data,but you must encapsulation operation with data, so you must add method "AddNewsComments" to the your repository and call this method in your manager.